src/applications/vaos/tests/appointment-list/components/AppointmentsPage/index.unit.spec.jsx

```diff
@@ -33,7 +33,7 @@ const initialState = {
   },
 };
 
-describe('VAOS integration: appointment list', () => {
+describe('VAOS <AppointmentsPage>', () => {
```

Comment:
I updated these test blocks to bring them back in line with most of our tests, which reference a page component